bundles/org.openhab.binding.twitter/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="twitter"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="account">
+		<label>Twitter Account</label>
+		<description>Use for sending Tweets</description>
+
+		<channels>
+			<channel id="lasttweet" typeId="lasttweet"/>
+		</channels>
+
+		<config-description>
+			<parameter name="consumerKey" type="text" required="true">
+				<label>Consumer API Key</label>
+				<context>password</context>
+			</parameter>
+			<parameter name="consumerSecret" type="text" required="true">
+				<label>Consumer API Secret</label>
+				<context>password</context>
+			</parameter>
+			<parameter name="accessToken" type="text" required="true">
+				<label>Access Token</label>
+				<context>password</context>
+			</parameter>
+			<parameter name="accessTokenSecret" type="text" required="true">
+				<label>Access Token Secret</label>
+				<context>password</context>
+			</parameter>
+			<parameter name="refresh" type="integer" required="false">
+				<label>Refresh Time for This Account in Mins</label>
```

Comment:
Labels are expected to be as short as possible. Guideline is 2-3 words with max 20-25 chars. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions
```suggestion
				<label>Refresh Time</label>
				<description>Time period between refreshs in minutes</description>
```